// orientlu test 1
syntax = "proto2";
option optimize_for = LITE_RUNTIME;

enum UserStatus {
    OFFLINE = 0;
    ONLINE = 1;
}

enum LogResult {
    LOGON_RESULT_SUCCESS = 0;
    LOGON_RESULT_NOTEXIST = 1;
    LOGON_RESULT_ERROR_PASSWD = 2;
    LOGON_RESULT_ALREADY_LOGON = 3;
    LOGON_RESULT_SERVER_ERROR = 4;
}

message UserInfo {
    required int64 acctID = 1;
    required string name = 2;
    required UserStatus status = 3;
}

message LogonRespMessage {
    required LogResult logonResult = 1;
    required UserInfo UserInfo = 2;
}
